hack/upstream-install.sh

```diff
@@ -0,0 +1,15 @@
+#! /bin/bash
+
+ROOT_DIR=$(dirname "${BASH_SOURCE}")/..
+source "${ROOT_DIR}/hack/common.sh"
+source "${ROOT_DIR}/hack/lib/customize-manifests.sh"
+
+set +u
+
+if [[ -z "$DISABLE_OCP_FEATURES" || "$DISABLE_OCP_FEATURES" = "false" ]]; then
+    INSTALLER_MANIFESTS_DIR="$OCP_INSTALLER_MANIFESTS_DIR"
+else
+    INSTALLER_MANIFESTS_DIR="$UPSTREAM_INSTALLER_MANIFESTS_DIR"
+fi
+
+source "${ROOT_DIR}/hack/install.sh"
```

Comment:
No `source`, we're executing the script. Also pass `"$@"` to the install.sh as args